### PR TITLE
[meta] first-timers-only issue template

### PR DESCRIPTION
**Pull request by tylerjw**
### Description

Here is an issue template for `first-timers-only` issues.  See: https://github.com/ros-planning/moveit/issues/2384

### CODE REVIEW

**Comment by jliukkonen** [comment]
I like this idea a lot. The template text feel welcoming as well. :)

**Comment by felixvd** [comment]
It is welcoming, but when I consider non-native speakers who might struggle with long English sentences, I think it would be good to make it simpler. Did you write it from scratch or is this based on something @tylerjw ?

**Comment by tylerjw** [comment]
It was largely copied from https://github.com/publiclab/plots2/issues/8646.  That is one of the many issues on the project from the author of the talk I listened to like this.  I'm happy to simplify it... are there specific sentances or sections that you think will be troublesome for people?

File: .github/ISSUE_TEMPLATE/first_timers_only.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+assignees: ''
+
+---
+
+## Background
+
+Overview of your issue here.
+
+## Instructions
+Hi, this is a `first-timer-only` issue. This means we've worked to make it more legible to folks who either **haven't contributed to our codebase before, or even folks who haven't contributed to open source before**.
```
Comment (felixvd) [inline .github/ISSUE_TEMPLATE/first_timers_only.md:15]:
```suggestion
Hi, this is a `first-timer-only` issue. This means we've worked to make it more legible to people who either **haven't contributed to our codebase before, or even folks who haven't contributed to open source before**.
```

File: .github/ISSUE_TEMPLATE/first_timers_only.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+If that's you, we're interested in helping you take the first step and can answer questions and help you out as you do. Note that we're especially interested in contributions from people from groups underrepresented in free and open-source software!
```
Comment (felixvd) [inline .github/ISSUE_TEMPLATE/first_timers_only.md:17]:
```suggestion
We're interested in helping you take the first step, and can answer questions and help you out along the way. Note that we're especially interested in contributions from underrepresented groups!
```

File: .github/ISSUE_TEMPLATE/first_timers_only.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+We know that the process of creating a pull request is the biggest barrier for new contributors. This issue is for you 💝
```
Comment (felixvd) [inline .github/ISSUE_TEMPLATE/first_timers_only.md:19]:
```suggestion
We know that creating a pull request is the biggest barrier for new contributors. This issue is for you 💝
```

File: .github/ISSUE_TEMPLATE/first_timers_only.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+
+If you have contributed before, **consider leaving this one for someone new**, and looking through our general [bug](https://github.com/ros-planning/moveit/labels/bug) issues. Thanks!
```
Comment (felixvd) [inline .github/ISSUE_TEMPLATE/first_timers_only.md:21]:
```suggestion
If you have contributed before, **consider leaving this PR for someone new**, and looking through our general [bug](https://github.com/ros-planning/moveit/labels/bug) issues. Thanks!
```

File: .github/ISSUE_TEMPLATE/first_timers_only.md [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+
+- [ ] 🗄️**Create a local workspace** for making your changes and testing [follwing these instructions](https://moveit.ros.org/install/source/)
+
+- [ ] 📝 **Update** the file(s) `$FILENAME`by applying the change shown in the diff below.
+
+```diff
+$DIFF
+```
+
+- [ ] 🤖 **Apply `clang-format-10`** auto formatting, [following these instructions](https://moveit.ros.org//documentation/contributing/code/?#clang-format-auto-code-formatting)
```
Comment (felixvd) [inline .github/ISSUE_TEMPLATE/first_timers_only.md:39]: This should include clang-tidy, now that I think about it.

File: .github/ISSUE_TEMPLATE/first_timers_only.md [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+
+- [ ] 💾 **Commit** your changes
+
+- [ ] 🔀 **Start a Pull Request**. There are two ways how you can start a pull request:
+1. If you are not familiar with GitHub or the pull request model, [here is a guide you can follow](https://guides.github.com/activities/hello-world/) on how GitHub works.
```
Comment (felixvd) [inline .github/ISSUE_TEMPLATE/first_timers_only.md:44]:
```suggestion
1. If you are not familiar with GitHub or how to create a pull request, [here is a guide you can follow](https://guides.github.com/activities/hello-world/) on how GitHub works.
```

**Review by felixvd — APPROVED** [review]
Thanks, I figured as much. I just suggested a few simplifications to avoid confusion (e.g. phrases that can have different meanings, long sentences or ambiguous relative pronouns). I'm happy with this otherwise.

File: .github/ISSUE_TEMPLATE/first_timers_only.md (hunk 5 above)
Comment (tylerjw) [reply .github/ISSUE_TEMPLATE/first_timers_only.md:39]: Good point, except we need to clean up our instructions for clang-tidy.  I couldn't figure out how to get it to work consistently following our instructions recently.  For now, my hope is that those of us who write these `first-timer-only` issues will either make sure the diff doesn't cause clang-tidy issues or help the newcomer with suggestions once it does in their PR.

**Review by sjahr — APPROVED** [review]
This sounds really good and it's a great idea reduce barrier for first-timers :+1: I sugested a few ideas for the template.

File: .github/ISSUE_TEMPLATE/first_timers_only.md [hunk 7]
```diff
@@ -0,0 +1,57 @@
...
+
+## Background
+
+Overview of your issue here.
+
+## Instructions
+Hi, this is a `first-timer-only` issue. This means we've worked to make it more legible to people who either **haven't contributed to our codebase before, or even folks who haven't contributed to open source before**.
+
+We're interested in helping you take the first step, and can answer questions and help you out along the way. Note that we're especially interested in contributions from underrepresented groups!
+
```
[the remark this answers is not in the document]
Comment (tylerjw) [reply .github/ISSUE_TEMPLATE/first_timers_only.md:18]: Two lines down I have a sentence that says this, would you rather this instead?

[the remark this answers is not in the document]
Comment (sjahr) [reply .github/ISSUE_TEMPLATE/first_timers_only.md:18]: No, this sounds very good. Sorry, I totally missed it.

**Pull request merged**